### ISSUE TITLE
Borg inconsistencies.

### ISSUE THREAD

**Issue by thebestcodertoevercode** [SPLURT-Station/S.P.L.U.R.T-Station-13#593]
## Round ID:
2335

## Testmerges:
#489 #427 #592 

## Reproduction:
This isn't really just a standard bug report that happens when you do a specific thing, but an issue with borgs in general.
During my time spent borging around, I've noticed a few issues that might have to be looked into. While these aren't game breaking at all, they do not feel intended.

### ERP related:

**- All flavor text (Including OOC text) reset.**
Each time I join the shift as a borg, it doesn't matter that I have my "Silicon Flavor Text" and "OOC Notes" filled out, it simply makes them blank, which forces me to use the IC tab to "Set Flavor Text" every shift.

**- No ERP sprites**
Regardless of what you set in your character prefs, turning into a borg completely negates any options and has you manually set a gender in the borg tab that appears topright. While this does give you the organs mechanically, they still lack any sprite and mechanically are actually quite lacking. I am aware that you also get a module that is a dildo, but it acts like holding a dildo, not a cock-- so even when you customize it and use it, it doesn't show up on the character's crotch.

**- Can't ejaculate and thus can't use the sex rack effectively**
With the limited cock/pussy borgs get, it makes the option of choosing what liquid your testicles create null and void. This also means making "messy finishes" is impossible-- and mechanically doesn't work on the research sex rack. It simply states that the target is not  human.
### Vore related:
**- Prey resisting** 
When an eaten prey squirms inside of a borg's vore panel (NOT the sleeper) it also doesn't show them any "Squirm messages (inside)" you set up, it only shows if they made any progress squirming (If your interactions are set to ON, that is). If you turn interactions off, every squirm is completely blank, **but** it does show the squirm messages you set to outside.

**- Examining** 
Much like the last issue, having any "Examine (outside)" does not seem to work at all, even if you set the required examine size to any percent.

**Comment by Zirok-BYOND** [comment]
Thanks for reporting stuff on Github!
> Flavor text resetting
Heck, this is still happening, huh. Odd. Might look into this.
> Expand module messes with layers
Huh.
> Broken pixel shifting
I'll check this out, I suppose.
> ERP Sprites
We have many dozen different borg sprites, you'd have to add these individually- A ton of work even if the individual sprites aren't that high effort.

**Comment by thebestcodertoevercode** [comment]
I understand the last part, it's more of an issue of having way too many origin points possible, and then you have the options of adding that to every single chassis (There's a lot of them) or have the player adjust it themselves (Which I presume is a lot of coding effort)

**Comment by thebestcodertoevercode** [comment]
If it's so easy to create, I wouldn't be complaining if you went ahead and made it, @Yawet330

**Comment by Zirok-BYOND** [comment]
@Yawet330 Yes, obviously. The point is that there are a billion very distinct chassis, each with their own location that genitalia ought to attach to.
